### ISSUE TITLE
support config file

### ISSUE THREAD

**Issue by arxdsilva** [pedrolopesme/call-it#5]
it'd be nice to have a optional config file that would let the user configure more things about the request. This can include:
- [x] multiple requests
- [x] body
- [x] headers

depends on #10 and #11 ;

**Comment by pedrolopesme** [comment]
great. All parameters currently supported by call-it could be passed via this config file.
